src/api/spec/models/status/report_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+RSpec.describe Status::Report, type: :model do
+  describe 'validations' do
+    it { is_expected.to validate_presence_of(:checkable) }
+
+    context 'for bs requests' do
+      let(:source_project) { create(:project_with_package) }
+      let(:target_project) { create(:project_with_package) }
+      let(:bs_request) do
+        create(:bs_request_with_submit_action,
+               source_project: source_project,
+               source_package: source_project.packages.first,
+               target_project: target_project,
+               target_package: target_project.packages.first)
+      end
+      let(:status_report) { create(:status_report, checkable: bs_request) }
+
+      it { expect(status_report).not_to validate_presence_of(:uuid) }
```

Comment:
is this needed? :thinking: I mean, you can test that it is valid.

Reply:
This validation should only run when the checkable is a Repository, see https://github.com/openSUSE/open-build-service/pull/5862/files#diff-b70f033bbb90cc314855c93d00984b60R20

Reply:
I know that! :stuck_out_tongue_winking_eye:  I meant that you could test that an status report is valid without `uuid` when it is a BsRequest instead of testing `not_to validate_presence_of`.

Reply:
![You say tomatoes](https://user-images.githubusercontent.com/514785/45804201-be280480-bcbb-11e8-963a-143a0584c4f2.gif)

@Ana06 you need to start tagging your nitpicks like we have agreed we would do. Please :smiley: